scripts/workspace_cromwell.py

```diff
@@ -0,0 +1,295 @@
+import os
+import json
+from pathlib import Path
+import requests
+import subprocess
+import sys
+import time
+import uuid
+
+
+# If a CROMWELL app exists return (app_name, status), otherwise (None, None).
+def check_for_app(env):
+    list_apps_url = f'{env["leonardo_url"]}/api/google/v1/apps/{env["google_project"]}'
+    r = requests.get(
+        list_apps_url,
+        params={
+          'includeDeleted': 'false'
+        },
+        headers = {
+            'Authorization': f'Bearer {env["token"]}',
+            'Referer': 'https://bvdp-saturn-dev.appspot.com/'
```

Comment:
In here and other functions, should this be pulled from `env` variable, (`env['referer']`)?

Reply:
Oops! Yes it definitely should!

Reply:
Although, actually, these aren't going through the proxy so I don't think they _need_ a referer at all